lib/auth/style.scss

```diff
@@ -194,4 +210,22 @@
   svg.icon-mail {
     color: $studio-gray-30;
   }
+  input {
+    border-color: $studio-gray-60;
+
+    &:focus {
+      border: 2px solid $studio-simplenote-blue-30;
+      outline: none;
```

Comment:
I was wondering if since some `border` properties are already defined in the default theme (light), here we could only have the border color as we have in the other states (disabled and read-only):
`border-color: $studio-simplenote-blue-30;`

And the same for `outline`, I think we could remove it from here too.

Reply:
Thanks! You're right that is a good call. Based on some other feedback I've switched to using box-shadow instead of border, and I don't believe there is a way to change only the color.
I did drop the outline though.